dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowAction.java

```diff
@@ -39,14 +40,49 @@
 	private String icon;
 	private boolean roleHierarchyForAssign;
 	private boolean requiresCheckout;
-	private String  requiresCheckoutOption;
 	private boolean assignable;
 	private boolean commentable;
 	private int order;
+	private Set<WorkflowStatus> showOn = Collections.emptySet();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Set to an EnumSet. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1641)